torch/onnx/_internal/fx/options.py

```diff
@@ -33,3 +40,10 @@ def update(self, **kwargs):
                     setattr(self, key, value)
             else:
                 raise KeyError(f"ExportOptions has no attribute {key}")
+
+        # NOTE(titaiwang): op_level_debug needs fixed shape to generate example inputs
+        # for torch ops and ONNX ops, but in dynamic export, we don't have this info, so
+        # op_level_debug would be forced to False if dynamic_axes is True
+        # https://github.com/microsoft/onnx-script/issues/393
+        if self.dynamic_axes and self.op_level_debug:
+            self.op_level_debug = False
```

Comment:
No. Let's assert with clear instruction. Implicitly changining user's direct input is not a good idea.
```
if op_level_debug:
  assert enable_dynamic_shape == False, "op_level_debug and enable_dynamic_shape are mutually execusive. Please set only one of them to be True"
  ```

Reply:
Done